sympy/physics/optics/utils.py

```diff
@@ -92,8 +107,39 @@ def refraction_angle(incident, medium1, medium2, normal=None, plane=None):
     [-sqrt(3)*sqrt(-2*n1**2/(3*n2**2) + 1)]])
     >>> refraction_angle(r1, n1, n2, plane=P)
     Ray3D(Point3D(0, 0, 0), Point3D(n1/n2, n1/n2, -sqrt(3)*sqrt(-2*n1**2/(3*n2**2) + 1)))
-
+    >>> round(refraction_angle(0.1, 1.2, 1.5), 5)
```

Comment:
perhaps that 0.1 could be changed to `rad(6)` to emphasize that the input is in radians
```python
Ray3D(Point3D(0, 0, 0), Point3D(n1/n2, n1/n2, -sqrt(3)*sqrt(-2*n1**2/(3*n2**2) + 1)))

The incident angle can also be given as a value in radians:

>>> from sympy import rad, pi
>>> round(refraction_angle(pi/30, 1.2, 1.5), 5)
0.08372
>>> round(refraction_angle(rad(6), 1.2, 1.5), 5)
0.08372
```